src/main/java/com/carleton/CapstoneSystem/models/Project.java

```diff
@@ -190,5 +193,13 @@ public void addStudentToAppliedList(Student student) {
     public void removeStudentFromAppliedList(Student student) {
         appliedStudents.remove(student);
     }
+
+    public Set<Submission> getProjectSubmissions() {
+        return projectSubmissions;
```

Comment:
it is not really practical to return a reference to your own set, maybe better to return a copy?